resources/l10n/bg/LC_MESSAGES/loot.po

```diff
@@ -127,140 +127,147 @@ msgstr "Мръсни приставки"
+#: src/gui/qt/main_window.cpp:599
 msgid "Search Cards..."
 msgstr "Търси в съдържанието..."
 
-#: src/gui/qt/main_window.cpp:594
+#: src/gui/qt/main_window.cpp:600
 msgid "Copy Load Order"
 msgstr "Копирай реда за зареждане"
 
-#: src/gui/qt/main_window.cpp:595
+#: src/gui/qt/main_window.cpp:601
 msgid "Copy Content"
 msgstr "Копирай съдържанието"
 
-#: src/gui/qt/main_window.cpp:596
+#: src/gui/qt/main_window.cpp:602
 msgid "Refresh Content"
 msgstr "Опресни съдържанието"
 
-#: src/gui/qt/main_window.cpp:597
+#: src/gui/qt/main_window.cpp:603
 msgid "Redate Plugins..."
 msgstr "Промени датата на приставките..."
 
-#: src/gui/qt/main_window.cpp:598
+#: src/gui/qt/main_window.cpp:604
 msgid "Clear All User Metadata..."
 msgstr "Изчисти всички добавени метаданни..."
 
-#: src/gui/qt/main_window.cpp:600
+#: src/gui/qt/main_window.cpp:606
 msgid "Plugin"
 msgstr "Приставка"
 
-#: src/gui/qt/main_window.cpp:601
+#: src/gui/qt/main_window.cpp:607
 msgid "Copy Metadata"
 msgstr "Копирай метаданните"
 
-#: src/gui/qt/main_window.cpp:602
+#: src/gui/qt/main_window.cpp:608
 msgid "Copy Card Content"
 msgstr "Копирай съдържанието"
 
-#: src/gui/qt/main_window.cpp:603
+#: src/gui/qt/main_window.cpp:609
 msgid "Edit Metadata..."
 msgstr "Редактирай метаданните..."
 
-#: src/gui/qt/main_window.cpp:604
+#: src/gui/qt/main_window.cpp:610
 msgid "Clear User Metadata..."
 msgstr "Изчисти добавените метаданни..."
 
-#: src/gui/qt/main_window.cpp:606
+#: src/gui/qt/main_window.cpp:612
 msgid "Help"
 msgstr "Помощ"
 
-#: src/gui/qt/main_window.cpp:607
+#: src/gui/qt/main_window.cpp:613
 msgid "View Documentation"
 msgstr "Виж документацията"
 
-#: src/gui/qt/main_window.cpp:608
-msgid "Open Debug Log Location"
-msgstr "Намери файла за регистриране на грешки"
-
-#: src/gui/qt/main_window.cpp:609
+#: src/gui/qt/main_window.cpp:614
 msgid "Join Discord Server"
 msgstr "Присъединете се към „Discord“"
 
-#: src/gui/qt/main_window.cpp:610
+#: src/gui/qt/main_window.cpp:615
 msgid "About"
 msgstr "Относно"
 
-#: src/gui/qt/main_window.cpp:613
+#: src/gui/qt/main_window.cpp:618
 msgid "Plugins"
 msgstr "Приставки"
 
-#: src/gui/qt/main_window.cpp:614
+#: src/gui/qt/main_window.cpp:619
 msgid "Filters"
 msgstr "Филтри"
 
-#: src/gui/qt/main_window.cpp:803 src/gui/qt/main_window.cpp:1708
+#: src/gui/qt/main_window.cpp:808 src/gui/qt/main_window.cpp:1759
 msgid "Updating and parsing masterlist..."
 msgstr "Обновяване и разбор на основния списък..."
 
-#: src/gui/qt/main_window.cpp:856
-#, c-format
+#: src/gui/qt/main_window.cpp:869
 msgid ""
-"This appears to be the first time you have run LOOT v%s. Here are some tips "
-"to help you get started with the interface."
+"This appears to be the first time you have run LOOT v%1%. Your current LOOT "
+"data has been backed up to: %2%"
 msgstr ""
-"Изглежда това е първото стартиране на „LOOT“ в%s. Ето някои съвети, които ще "
-"са Ви от полза."
+"Изглежда това е първото стартиране на „LOOT“ в%s. Текущите данни на „LOOT“ "
```

Comment:
You missed a `%s` getting replaced here.

Reply:
Fixed.